polar-core/src/vm.rs

```diff
@@ -357,8 +360,7 @@ impl PolarVirtualMachine {
                 call_id,
                 instance,
                 field,
-                check_errors,
```

Comment:
Previously we needed to push a check error goal after every new LookupExternal event we created. Since there is only ever one LookupExternal event, the CheckError goal can just be pushed directly by the goal creator.

Reply:
Which is what is happening here: https://github.com/osohq/oso/pull/485/files#diff-31c6da6cb2dee04c7bdefb96252140c683d8d697de90ec8ae7a6745bf46139d9R1635